From 55c32e6cb74b017b27846aa276b64d9bdc5f556a Mon Sep 17 00:00:00 2001
From: Benoit Bovy <benbovy@gmail.com>
Date: Wed, 26 Feb 2025 12:28:49 +0100
Subject: [PATCH] fix-cmake

---
 CMakeLists.txt | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index f02a7e5..c347d40 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -48,7 +48,7 @@ message(STATUS "Found numpy v${Python_NumPy_VERSION}: ${Python_NumPy_INCLUDE_DIR
 find_package(pybind11 REQUIRED)
 message(STATUS "Found pybind11 v${pybind11_VERSION}: ${pybind11_INCLUDE_DIRS}")
 
-find_package(s2 REQUIRED)
+find_package(s2 CONFIG REQUIRED)
 
 # Build
 # =====
@@ -65,7 +65,7 @@ target_link_libraries(pys2index PRIVATE
   xtensor-python
   pybind11::module
   Python::NumPy
-  s2
+  s2::s2
   )
 
 pybind11_extension(pys2index)
-- 
2.43.0

