diff -ruN -x __pycache__ -x obj -x '*~' -x tags -x TAGS -x autom4te.cache pys2index-0.1.3.orig/setup.py pys2index-0.1.3/setup.py
--- pys2index-0.1.3.orig/setup.py	2023-07-10 15:11:55.081484256 +0200
+++ pys2index-0.1.3/setup.py	2023-07-10 15:19:27.957633538 +0200
@@ -40,10 +40,11 @@
             extdir += os.path.sep
 
         cmake_args = ['-DCMAKE_LIBRARY_OUTPUT_DIRECTORY=' + extdir,
-                      '-DPYTHON_EXECUTABLE=' + sys.executable,
                       '-DPython3_EXECUTABLE=' + sys.executable,
                       '-DPython3_NumPy_INCLUDE_DIR=' + np.get_include()]
 
+        cmake_args += os.environ.get("CMAKE_ARGS", "").split(" ")
+
         cfg = 'Debug' if self.debug else 'Release'
         build_args = ['--config', cfg]
 
